### DISCUSSION TITLE
Which puppeteer version for codeceptjs?

### DISCUSSION THREAD

**Discussion by incoming-th** [codeceptjs/CodeceptJS#3800] [General]
Hi,

With each update of Codeceptjs I always ask myself if I should update the puppeteer version or not?

How to get this information?

For example, in GIT I see it's ^10 but now puppeteer is at version ^21 on their repo.

Thanks

**Comment by kobenguyent** [comment]
Usually, I think you just need to go with version which is defined by codeceptjs.
Btw, we just have a PR to update the pupeeteer to latest version https://github.com/codeceptjs/CodeceptJS/pull/3856
